frontend/src/components/Forms/TeamEventCreditsForm/TeamEventsCreditDashboard.tsx

```diff
@@ -28,6 +35,34 @@ const TeamEventCreditDashboard = (props: {
     return 1;
   };
 
+  const deleteTECAttendanceRequest = (attendance: TeamEventAttendance) => {
+    // const deleteTECAttendanceRequest = () => {
+
+    TeamEventsAPI.deleteTeamEventAttendance(attendance.uuid)
+      .then(() => {
+        setShowComponent(false);
+
+        Emitters.generalSuccess.emit({
+          headerMsg: 'Team Event Attendance Deleted!',
+          contentMsg: 'Your team event attendance was successfully deleted!'
+        });
+        ImagesAPI.deleteEventProofImage(attendance.image);
+        Emitters.teamEventsUpdated.emit();
+      })
+      .catch((error) => {
+        Emitters.generalError.emit({
+          headerMsg: 'You are not allowed to delete this team event attendance!',
+          contentMsg: error
+        });
+      });
+  };
+
+  const [showComponent, setShowComponent] = useState(true);
+
+  const handleRemoveComponent = (attendance: TeamEventAttendance) => {
```

Comment:
Delete this function `handleRemoveComponent`, we can handle all the logic that goes into deleting and updating the UI in the `deleteTECAttendanceRequest` function.